Jenkinsfile

```diff
@@ -19,7 +19,7 @@ def images = [
     [image: "balenalib/rpi-raspbian:stretch", arches: ["armhf"]],
     [image: "balenalib/rpi-raspbian:buster",  arches: ["armhf"]],
     [image: "ubuntu:xenial",                  arches: ["amd64", "aarch64", "armhf"]], // Ubuntu 16.04 LTS (End of support: April, 2021. EOL: April, 2024)
-    [image: "ubuntu:bionic",                  arches: ["amd64", "aarch64", "armhf"]], // Ubuntu 18.04 LTS (End of support: April, 2023. EOL: April, 2028)
+    [image: "ubuntu:bionic",                  arches: ["amd64", "aarch64", "armhf", "s390x"]], // Ubuntu 18.04 LTS (End of support: April, 2023. EOL: April, 2028)
     [image: "ubuntu:disco",                   arches: ["amd64", "aarch64", "armhf"]], // Ubuntu 19.03  (EOL: January, 2020)
     [image: "ubuntu:eoan",                    arches: ["amd64", "aarch64", "armhf"]], // Ubuntu 19.10  (EOL: July, 2020)
     [image: "ubuntu:focal",                   arches: ["amd64", "aarch64"]],          // Ubuntu 20.04 LTS (End of support: April, 2025. EOL: April, 2030)
```

Comment:
Should we enable it for all ubuntu versions?

Reply:
Good catch. That was for the first test builds. I'll add the others as well.

Reply:
Added it to debian and ubuntu versions.